### PR TITLE
Update Mobile Verify step translations

### PR DESCRIPTION
**Pull request by nenadjelovac**

### CODE REVIEW

File: locales/en.yml [hunk 1]
```diff
@@ -460,6 +460,20 @@ en:
     placeholder: 6-Digit Verification Code
     subtitle: One-time anonymous text message.
     title: Verify Mobile Number
+  signup_verify_email:
```
Comment (nenadjelovac) [inline locales/en.yml:463]: @nielsenramon I added these missing translations for Verify Email step while we're at it. Mind reviewing the string names and copy before I apply it to other translation files as well 🙏

File: locales/en.yml [hunk 2]
```diff
@@ -452,14 +452,28 @@ en:
...
     title: Verification Time!
   signup_verify:
-    cta: Complete Signup & Verification
+    cta: Verify Mobile Number
     incorrect_code: "<strong>Kinky Operator:</strong> Doh&hellip; the verification code you entered seems off. Please verify that the number above matches what you received from us and try again."
-    label: Verification Code
-    notice: We sent the verification code to <strong>%{phone_number}</strong>. If after a couple minutes you don't receive a text message from us, <a class="silver" href="%{back_path}">update your number and we'll send you a new verification code</a>.
-    page_title: Enter Verification Code
+    label: 6-digit Verification Code
+    notice: We sent the verification code to <strong>%{phone_number}</strong>. If after a couple minutes you don't receive a text message from us, <a class="color-inherit" href="%{back_path}">update your number and we'll send you a new verification code</a>.
```
Comment (nielsenramon) [inline locales/en.yml:458]: we'll need a `title` on that links

File: locales/en.yml [hunk 3]
```diff
@@ -452,14 +452,28 @@ en:
...
+    page_title: Verify Mobile Number
     placeholder: 6-Digit Verification Code
-    subtitle: That was a bit painful wasn't it?!?!
-    title: Last Step
+    subtitle: One-time anonymous text message.
```
Comment (nielsenramon) [inline locales/en.yml:461]: `You'll receive a one-time anonymous text message.`

File: locales/en.yml [hunk 4]
```diff
@@ -452,14 +452,28 @@ en:
...
+    title: Verify Mobile Number
```
Comment (nielsenramon) [inline locales/en.yml:462]: page_title and title. What is the difference here?

File: locales/en.yml [hunk 5]
```diff
@@ -452,14 +452,28 @@ en:
...
+  signup_verify_email:
+    cta: Verify Email Address
+    footnote: By clicking "Verify Email Address" you agree to respect and follow our <a class="gray-400 link underline hover-gray-300" href="%{tou_path}" target="_blank">Terms of Use</a>, <a class="gray-400 link underline hover-gray-300" href="%{privacy_path}" target="_blank">Privacy Policy</a>, and <a class="gray-400 link underline hover-gray-300" href="%{guidelines_path}" target="_blank">Community Guidelines</a>.
+    label: 6-digit Verification Code
+    page_title: Verify Email Address
+    subtitle: You're just one step away from perving FetLife.
```
Comment (nielsenramon) [inline locales/en.yml:468]: `You're just one step away from perving FetLife!`

File: locales/en.yml [hunk 6]
```diff
@@ -452,14 +452,28 @@ en:
...
+    title: Verify Email Address
+  signup_change_email:
```
Comment (nielsenramon) [inline locales/en.yml:470]: Is this the copy when you change the email address you entered during signup?

Comment (nenadjelovac) [reply locales/en.yml:470]: @nielsenramon yes!

File: locales/en.yml (hunk 4 above)
Comment (nenadjelovac) [reply locales/en.yml:462]: @nielsenramon Good question, I don't know. I'll check if `page_title` falls back to `title` if nothing is defined for `page_title`, and if so I'll just define the `title`.

Comment (nenadjelovac) [reply locales/en.yml:462]: @nielsenramon I realized I didn't answer your question 🤦  😆  The difference is that the `page_title` is the title in browser tab (`<title>` tag), and `title` is the red heading on the page.

I checked if `page_title` falls back to `title`, and it doesn't actually do that. 

It looks like first three pages of the signup flow use the same title `Join` (`signup.title`). Should I leave it like that on those pages?

File: locales/en.yml (hunk 6 above)
Comment (nenadjelovac) [reply locales/en.yml:470]: @nielsenramon looks good?

File: locales/en.yml [hunk 7]
```diff
@@ -449,17 +449,31 @@ en:
...
+    label: 6-digit Verification Code
+    notice: We sent the verification code to <strong>%{phone_number}</strong>. If after a couple minutes you don't receive a text message from us, <a class="color-inherit" href="%{back_path}" title="update your number">update your number and we'll send you a new verification code</a>.
+    page_title: Verify Mobile Number
     placeholder: 6-Digit Verification Code
-    subtitle: That was a bit painful wasn't it?!?!
-    title: Last Step
+    subtitle: You'll receive a one-time anonymous text message.
+    title: Verify Mobile Number
+  signup_verify_email:
+    cta: Verify Email Address
```
Comment (nielsenramon) [inline locales/en.yml:464]: Should we say: `Verify & Complete Signup`?

Is there any way I can see screenshots for these screens? Would love to review them on a screenshot 😬

Comment (nenadjelovac) [reply locales/en.yml:464]: @nielsenramon perhaps.

Good idea, here are the screenshots:

Screen Shot 2022-02-10 at 15 58 28
Screen Shot 2022-02-10 at 15 58 42
Screen Shot 2022-02-10 at 16 03 01

Comment (nielsenramon) [reply locales/en.yml:464]: @nenadjelovac Mind posting these on the issue? I think we need to clean some things up before we move forward with these changes :-/

Comment (nenadjelovac) [reply locales/en.yml:464]: @nielsenramon sure!

**Pull request closed without merging**